### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by ahoffer** [gilesknap/gphotos-sync#371] [Q&A]
gphotos-sync worked really well for me. 
I don't have any code to contribute. 
Is there a place I can donate some money to the project?

**Comment by gilesknap** [comment]
Hi @ahoffer  thanks for the offer. This project is Free Open Source Software. No need for donations, I enjoy making things and having people find them useful.

**Reply by Koramchad** [reply]
That's really nice of you, but I would like to second ahoffer's message. Do you have a bitcoin/paypal address where we can send you a little token of our gratitude?

**Reply by gilesknap** [reply]
Thanks for the appreciation but no funds necessary!
